### PR TITLE
Allow controlling badge margin via mixin

### PR DESCRIPTION
**Pull request by andrey-git**
* Allow controlling badge margin via mixin.
* Minor fix for ha-cards which was reserving extra title space for cards with at least 2 entities, where it should be reserved for 1-entity cards too.

Full disclosure: I'm going to use the margin fix in custom UI to support putting badges in the state cards.

### CODE REVIEW

File: src/components/ha-label-badge.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
       display: inline-block;
       text-align: center;
       vertical-align: top;
-      margin-bottom: 16px;
```
Comment (balloob) [inline src/components/ha-label-badge.html]: I think that the badge should never had a margin built-in to begin with. A margin on the outside should be part of whatever holds the label badge

**Comment by balloob** [comment]
Bump 💃

**Comment by andrey-git** [comment]
Will get to this this week.

**Comment by andrey-git** [comment]
Done

File: src/cards/ha-badges-card.html [hunk 2]
```diff
@@ -4,6 +4,12 @@
 
 <dom-module id='ha-badges-card'>
   <template>
+    <style>
+      ha-state-label-badge {
+        display: inline-block;
+        margin-bottom: var(--ha-state-label-badge-margin-bottom, 16px);
```
Comment (balloob) [inline src/cards/ha-badges-card.html]: Same idea, does this card need to define the margin? 😛  feels like something that should be done by the component that actually cares about this margin.

**Comment by andrey-git** [comment]
Since the badges can take more than 1 line the margin has to go on `ha-state-label-badge`
I'm not aware of a CSS I can give to `ha-badges-card` to tell it "Have some margin between non-text 'lines' of divs"

**Comment by balloob** [comment]
Alrighty!

**Pull request merged**